pom.xml

```diff
@@ -55,11 +55,49 @@
 
   <dependencies>
     <dependency>
+     <groupId>org.jenkins-ci.main</groupId>
+     <artifactId>jenkins-core</artifactId>
+     <version>2.63</version>
+     <scope>provided</scope>
+    </dependency>
+    <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>dashboard-view</artifactId>
       <version>2.0</version>
       <optional>true</optional>
     </dependency>
+    <!-- dependencies on Jenkins Pipeline plugins -->  
+    <dependency>  
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>  
+      <artifactId>workflow-job</artifactId>  
+      <version>1.14</version>  
```

Comment:
I have developed new step (publishSloccount) thus dependency to workflow-step-api, workflow-job and workflow-cps is needed.
workflow-support and workflow-basic-steps are indeed for test purpose only.